### PR TITLE
Abous us section dark mode changes

### LINKED ISSUE #495: [BUG] About us page dark mode.
### Description

Nothing is clearly visible in dark mode of about us page.

### Screenshots

Screenshot 2024-10-16 at 8 10 15 PM

### Any additional information?

_No response_

### What browser are you seeing the problem on?

Chrome

### Checklist

- [X] I have checked the existing [issues](https://github.com/swaraj-das/Collect-your-GamingTools/issues).
- [X] I have read the [Code Of Conduct](https://github.com/swaraj-das/Collect-your-GamingToolst/blob/main/CODE_OF_CONDUCT.md)
- [X] The changes don't break the code
- [X] Please attach the screenshots and video of the development work once it is done

### PR DESCRIPTION
**Pull request by saumyayadav25**
# Pull Request Format
## PR Title

Fixes #495 

## Type of PR
- [X] Bug fix
- [X] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): ___________  

## Description
I created a issue regarding content not visible in dark mode, but it was already solved by someone else. but still there were some issues:
1) cart black image used-> not visible in dark mode, even if there are changes in future, we just can't change image everytime.
2) Redundancy of content.
## Screenshots / Videos (if applicable)
**Before:**
Screenshot 2024-10-18 at 5 41 59 PM

**After:**
- added cart fa which can be changed anytime if in case theme of the website changes.
- Redundant part was removed.
Screenshot 2024-10-18 at 5 42 08 PM

## Checklist
- **Add X in the box to specify.**
- [X] I have performed a self-review of my code.
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### Additional Context
I've noticed that people are just copy and pasting the code instead of making changes themselves. which lead to redundancy of content which can be even seen in before image.
When i created the issue, the redundant content was not there.

Thank you for reviewing my pull request!

### CODE REVIEW

**Comment by saumyayadav25** [comment]
@swaraj-das kindly Review this PR

**Comment by saumyayadav25** [comment]
@swaraj-das kindly review this PR

**Comment by swaraj-das** [comment]
@saumyayadav25 This has already been fixed by other contributors. Please find another issue to contribute to.

File: aboutus.html [hunk 1]
```diff
@@ -220,7 +221,7 @@ <h1>About <span> Gaming </span>Tools</h1>
             <h2 class="h2">Why Choose Us?</h2>
         </div>
 
-        
+        <h2>Why Choose Us</h2>
```
Comment (swaraj-das) [inline aboutus.html:224]: Remove this line.

**Comment by swaraj-das** [comment]
@saumyayadav25 Your code makes the cart button white, it's not the solution. Please copy the code of cart button from index.html and fix here in aboutUs.html

**Comment by saumyayadav25** [comment]
> @saumyayadav25 Your code makes the cart button white, it's not the solution. Please copy the code of cart button from index.html and fix here in aboutUs.html

okay

**Comment by saumyayadav25** [comment]
@swaraj-das this PR can be closed as about-us section changes are implemented by someone else. if you need any changes in the current about-us section u can let me know.

**Pull request closed without merging**